pkg/server/chaosd/network.go

```diff
@@ -194,13 +209,75 @@ func (s *Server) applyTC(attack *core.NetworkCommand, ipset string, uid string)
 	return nil
 }
 
+func (s *Server) applyEtcHosts(attack *core.NetworkCommand, uid string) error {
+	backupCmd := exec.Command("/bin/bash", "-c", "mv /etc/hosts /etc/hosts.chaosd && touch /etc/hosts")
+	if err := backupCmd.Run(); err != nil {
```

Comment:
it is better to update to `backupCmd.Output`, so we can get some useful information if it runs failed